src/tracing/tracing.js

```diff
@@ -32,8 +32,9 @@ export default class Tracing {
     return {
       attributes: {
         ...(this.options.resource || {}),
-        'rollbar.environment': this.options.environment,
-      }
+        'rollbar.environment':
+          this.options.payload?.environment ?? this.options.environment,
```

Comment:
This was added in `ES2020`, and i see all the optional chaining operators are added in replay. I assume this is fine because there is [this notion doc](https://www.notion.so/rollbar/Changes-to-Rollbar-js-build-targets-1c2510a20ff680bf8ab7eb387e34a1b6?pvs=4) for targeting `ES2021`?

Reply:
Correct! Already checked that with @waltjones.